### ISSUE TITLE
submit upstream

### ISSUE THREAD

**Issue by Be-ing** [22degrees/RoundCorners#34]
This skin looks nice. It would be great to include it in Mixxx. I do not think we have time to do a thorough review of a whole new skin for Mixxx 2.1, but it would be good to get it merged upstream for Mixxx 2.2. To submit it for inclusion, fork the [mixxxdj/mixxx repository](https://github.com/mixxxdj/mixxx), clone your forked repository to your computer, add this skin as a new folder in the res/skins folder, commit it on a new branch, and push. Then make a pull request for the mixxxdj/mixxx repository. It's fine if the skin is not complete yet, just make sure to say so in the pull request. Having a work-in-progress pull request upstream will let more people test the skin and give feedback.

**Comment by wiezo** [comment]
I agree with the proposal of Be. Your skin is very useful on a 4K screen!
Thanks for the work involved!
